Code/GraphMol/RGroupDecomposition/RGroupDecomp.cpp

```diff
@@ -496,7 +597,7 @@ struct RGroupDecompData {
     size_t idx = 0;
     for (auto coreIt = cores.begin(); coreIt != cores.end(); ++coreIt, ++idx) {
       RWMol *alignCore = coreIt->first ? &cores[0] : nullptr;
-      params.prepareCore(coreIt->second, alignCore);
+	  PRECONDITION(params.prepareCore(coreIt->second, alignCore), "Could not prepare at least one core");
```

Comment:
This isn't really a PRECONDITION. 
To be consistent it should be `CHECK_INVARIANT`